rsrcdump/structtemplate.py

```diff
@@ -108,7 +113,7 @@ def tag_values(self, values: tuple):
             # Multiple-element structure but no field names: return the tuple
             return values
 
-    def pack(self, obj: Any) -> bytes:
+    def pack(self, obj: list[str] | str) -> bytes:
```

Comment:
`obj` can be a JSON blob. I want to say Any here

Reply:
```
rsrcdump/structtemplate.py:121:37: error: Argument 1 to "pack_record" of "StructTemplate" has incompatible type "object"; expected "list[str] | dict[str, str] | str"  [arg-type]
```

Reply:
Updated to be more accurate in e907f63